config/locales/translations/zendesk_apps_support.yml

```diff
@@ -175,6 +178,12 @@ parts:
       key: "txt.apps.admin.error.app_build.no_code_for_ifo_notemplate"
       title: "App builder job: code included for a type of app that shouldn't have code"
       value: "Javascripts, stylesheets, and templates are not allowed when an iframe URI or noTemplate is specified"
+      obsolete: "2019-11-01"
+  - translation:
+      key: "txt.apps.admin.error.app_build.no_source_required_apps"
+      title: "Inform users that no source files are required when uploading marketingOnly and requirementsOnly apps. marketingOnly and requirementsOnly are Manifest properties and always English. https://developer.zendesk.com/apps/docs/developer-guide/manifest Also refer https://en.wikipedia.org/wiki/Style_sheet_(web_development) for stylesheets."
```

Comment:
This title field is now updated with the link to Manifest properties (marketingOnly and requirementsOnly) and a wiki-page to "stylesheets".

Thank you for reviewing, @hiroiguchi .